src/modules/node-restore.ts

```diff
@@ -3,20 +3,29 @@ import shell from "shelljs";
 import { CommandStringBuilder } from "../utilities/command-string-builder";
 import { OptionStringBuilder } from "../utilities/option-string-builder";
 import { Echo } from "./echo";
+import { Options } from "../constants/options";
+
+// -----------------------------------------------------------------------------------------
+// #region Constants
+// -----------------------------------------------------------------------------------------
+
+const COMMAND = new CommandStringBuilder("npm", "install");
```

Comment:
Just something to consider here.  The way this is written, every time `COMMAND` is executed somewhere within the context of a single command execution, this will generate a new `CommandStringBuilder` which, albeit a minor lift, isn't the most efficient means of execution.  Suggest defining the new `CommandStringBuilder` in a line before to avoid multiple executation.

Reply:
Ignore this comment.  I was wrong.  Thought was being exporte directly originally.